Exercise_C.js

```diff
@@ -14,5 +14,14 @@ See README.md if you need help running your code.
 */
 
 for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+  if(i%3 == 0 && i%5 == 0)
+  {
+    console.log("Hong Kong");
+  }else if(i%3 == 0)
+  {
+    console.log("Hong");
+  }else if(i%5 == 0)
+  {
+    console.log("Kong");
+  }
 }

```

Comment:
For this question, it would be better to use this as in your for loop you have used <= instead of < 

```
for (let i = 1; i < 30; i++) {
  if (i % 3 == 0 && i % 5 == 0) {
    console.log("Hong Kong");
  } else if (i % 3 == 0) {
    console.log("Hong");
  } else if (i % 5 == 0) {
    console.log("Kong");
  } else {
    console.log(i);
  }
}
```
Or you can try this one 
```
for (let i = 1; i <= 30; i++) {
  if (i % 15 == 0) {
    console.log("Hong Kong");
  } else if (i % 3 == 0) {
    console.log("Hong");
  } else if (i % 5 == 0) {
    console.log("Kong");
  } else {
    console.log(i);
  }
}
```